techniques/applications/packageManagement/1.2/metadata.xml

```diff
@@ -3,7 +3,7 @@
   <DESCRIPTION>This technique operates on individual packages.
 
   It will ensure that the defined packages are in the desired state (installed or removed) using the appropriate package manager.</DESCRIPTION>
-  
+  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next stable version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```

Comment:
```suggestion
  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```